lighthouse-plugin-publisher-ads/package.json

```diff
@@ -23,7 +33,7 @@
     "yargs": "^13.3.0"
   },
   "peerDependencies": {
-    "lighthouse": "^5.5.0"
+    "lighthouse": "^5.6.0"
```

Comment:
might want to do `>=5.6.0` for the plugin package.json? Hopefully wouldn't need that forever but may avoid some unpleasant juggling when Lighthouse does ship 6.0/6.0.0-secret-alpha or whatever

(and theoretically future versions will be compatible-ish :)